x-pack/plugins/ml/public/file_datavisualizer/components/import_view/import_progress.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React from 'react';
+
+import {
+  EuiStepsHorizontal,
+  EuiProgress,
+  EuiSpacer,
+} from '@elastic/eui';
+
+export const IMPORT_STATUS = {
+  INCOMPLETE: 'incomplete',
+  COMPLETE: 'complete',
+  FAILED: 'danger',
```

Comment:
Why is this not mapped this to `DANGER`? Is it worth moving these to `common/constants/states.js` ?

Reply:
although the class is `danger`, the status is really `failed`, so named it as such.
The names of the constants are more import than the values, and `failed` is clearer IMO